mass_editing/models/mass_editing_line.py

```diff
@@ -38,6 +38,12 @@ class MassEditingLine(models.Model):
         "'many2many_tags', 'selection'",
     )
 
+    apply_domain = fields.Boolean(
+        default=True,
```

Comment:
Please, we don't want to fail by default

```suggestion
        default=False,
```